lib/mongodbQueue.js

```diff
@@ -99,6 +99,7 @@ MongodbQueue.prototype.create = function(cb) {
             } else if (existingIndex.expireAfterSeconds !== self.queueOptions.ttl) {
               return callback(null, true);
             }
+            return callback(null, true);
```

Comment:
@aidenkeating I think `false` makes sense as the default here. The index should only be dropped if it exists and the the expiry value is different than the configured value.
Does that make sense?

We could probably simplify the condition slightly then too.
e.g.
```js
if (existingIndex && existingIndex.expireAfterSeconds !== self.queueOptions.ttl) {
  return callback(null, true);
} else {
  return callback(null, false);
```